File: python/basics/files.py
# read from file
with open('path/to/file', 'r') as file:
    file.read()

# or
content = open('path/to/file', 'r').read()

# read lines
with open('path/to/file.txt', 'r') as file:
    for line in file.readlines():
        # do something

# write to file
with open('path/to/file.txt', 'w') as file:
    file.write('This is some text to write.')

File: _scala/primer/files.scala
import scala.io.Source
import java.io._

// read from file
val file = Source.fromFile("path/to/file")

for (line <- file.getLines) {
    // do something
}
file.close

// write to file
val file = new File("path/to/file.txt")
val buffer = new BufferedWriter(new FileWriter(file))

buffer.write("Text.")
buffer.close